chart/monocular/templates/repo-sync-cronjobs.yaml

```diff
@@ -19,7 +19,16 @@ spec:
     spec:
       template:
 {{ include "monocular.sync.podTemplate" (list . $) | indent 8 }}
-  schedule: "0 * * * *"
-  successfulJobsHistoryLimit: 3
+  schedule: {{ if not .schedule -}}
```

Comment:
I think this can be simplified to a one-liner with `{{ default "0 * * * *" .schedule | quote }}`

Reply:
wow, this is awesome. I did not know sprig had a default function.